meta_capstone/ViewControllers/InGameViewController.m

```diff
@@ -484,28 +535,28 @@ - (void) createTilesTesting: (NSString *)word : (int) xIndex : (int) yIndex : (B
 }
 
 - (void) createBoard: (NSArray *)words {
-    //empty array, remove later
-    Tile *empty = [[Tile alloc] init];
-    empty.fillable = NO;
-    NSMutableArray *innerArray = [[NSMutableArray alloc] initWithCapacity: 10];
-    for (int j = 0; j < 10; j++) {
-        [innerArray insertObject:empty atIndex:j];
-    }
+//    //empty array, remove later
+//    Tile *empty = [[Tile alloc] init];
+//    empty.fillable = NO;
+//    NSMutableArray *innerArray = [[NSMutableArray alloc] initWithCapacity: 10];
+//    for (int j = 0; j < 10; j++) {
+//        [innerArray insertObject:empty atIndex:j];
+//    }
+//    //add inner array to tilesArray
+//    self.tilesArray = [[NSMutableArray alloc] initWithCapacity: 10];
+//    for (int i = 0; i < 10; i++) {
+//        [self.tilesArray insertObject:[NSMutableArray arrayWithArray:innerArray] atIndex:i];
```

Comment:
can you explain why you commented out and left this here?

Reply:
i used the xcode tests to run just this method while working on the algorithm, so i needed to initialize the board within this method. all the commented code (methods with "Testing") was an alternate version i added so i could keep working on the algorithm without querying parse every time, so i left it in while i was still working and removed it once i felt the algorithm was in a decent place.